app/api.py

```diff
@@ -448,6 +448,51 @@ def channel_new_channel(self, id):
 
         return Response(json.dumps(data, sort_keys=True, indent=4), mimetype='application/json')
 
+    @route('<int:id>/channels/<int:channel_id>', methods=['PATCH'])
+    def updates_channel(self, id, channel_id):
+        """ Update specific channel attributes
+        """
+
+        server = meta.getServer(id)
+
+        # Return 404 if not found
+        if server is None:
+            return jsonify(message="Not Found"), 404
+
+        # Return 404 if not found
+        channel = server.getChannelState(channel_id)
+        if channel is None:
+            return jsonify(message="Not Found"), 404
+
+        name = request.form.get('name')
+        parent = request.form.get('parent')
+        description = request.form.get('description')
+        links = request.form.getlist('links[]')
+        description = request.form.get('description')
+        position = request.form.get('position')
+        temporary = request.form.get('temporary')
+
+        if name is not None:
+            channel.name = name.encode('utf-8')
```

Comment:
Hey, have you tried running this with the latest from `master`? I tried passing `name` in the form and it errors when trying to set the channel state. It works if I remove the `.encode('utf-8)` part, though.

Reply:
oh
If don't use `.encdoe('utf-8')`, name will be `{}` in API response.
But name is ok on mumble client.

So I just try to make response correct value.

Reply:
I try to use `str(name)`, after updating will `getChannelState(channel_id)` again for response.
It works.